extensions/oauth/src/test/java/org/mule/test/oauth2/AbstractOAuthAuthorizationTestCase.java

```diff
@@ -91,14 +98,21 @@
   @Rule
   public SystemProperty wireMockHttpPort = new SystemProperty("oauthServerHttpPort", String.valueOf(oauthServerPort.getNumber()));
 
+  private ExtensionsClient client;
+
   @Before
   public void before() throws Exception {
     try {
-      // Force the initialization of the OAuth context
-      // TODO MULE-11405 switch to the client
-      // muleContext.getRegistry().lookupObject(ExtensionsClient.class).execute("HTTP", "request",
-      // builder().configName("requestConfig").build());
-      flowRunner("testFlow").runNoVerify();
+      //Force the initialization of the OAuth context
+      client = muleContext.getRegistry().lookupObject(ExtensionsClient.class);
+      client.execute("HTTP", "request",
+                     DefaultOperationParameters.builder()
+                         .configName("requestConfig")
+                         .addParameter("host", "localhost")
```

Comment:
are the parameters needed even if they are already in the config in the application?